components/scream/src/control/atmosphere_driver.cpp

```diff
@@ -802,6 +802,11 @@ void AtmosphereDriver::run (const int dt) {
   // Make sure the end of the time step is after the current start_time
   EKAT_REQUIRE_MSG (dt>0, "Error! Input time step must be positive.\n");
 
+  // Print current timestamp information
```

Comment:
Probably worth putting in the PR description that you also add a timestamp to typical output.  Might be worth having this be a runtime option, it would be pretty straightforward to have a YAML key for it that defaults to TRUE.

Reply:
Yeah sure. I could also remove it if you think it doesn't belong here.

Reply:
Imho, this can be addressed when tackling the output logging. Rather than add a yaml entry for each piece of information, I think it will be easier to add a "log level" option, and internally we decide what level is required to output each piece of information.

Reply:
@bartgol @AaronDonahue do you want me to remove this for now then? I find it pretty useful to see what step the model made it to without looking at the output netcdf files, which first requires hand configuring that output.

Reply:
I think you can keep it where it is.

Reply:
agreed, we can always replace when we get more robust logging in place